ngraph/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # ******************************************************************************
 
-cmake_minimum_required (VERSION 3.11)
+cmake_minimum_required (VERSION 3.13)
```

Comment:
looks like we don't need it for sub folder. It's enough to have only in root cmake for OpenVINO (plus for python, since it can be used separately)

Reply:
And in template plugin as well, I think.

I updated to 3.13 everywhere except for 3rdparty folders - just to be consistent and not to confuse users why certain parts require different versions.

Reply:
Let's remove it for ngraph. It's enough to have only in root cmake for OpenVINO